content/20.pharmaceuticals.md

```diff
@@ -27,54 +27,56 @@ The novel coronavirus _Severe acute respiratory syndrome-related coronavirus 2_
+While a number of vaccines have been developed and approved in different countries starting in late 2020 [@individual-vaccines], vaccination efforts have not proceeded at the same pace throughout the world and are not yet close to ending the pandemic.
 
 Due to the continued threat of the virus and the severity of the disease, the identification and development of therapeutic interventions have emerged as significant international priorities.
-Prior developments during other recent outbreaks of emerging diseases, especially those caused by human coronaviruses (HCoV), have provided a number of hypotheses guiding a biomedical approach to the behavior and treatment of this novel coronavirus infection.
-However, previous emerging HCoV-related disease threats have been controlled much more quickly than SARS-CoV-2 through public health efforts (Figure @fig:csse-deaths), which has made the repurposing and development of pharmaceuticals uniquely important in the face of this crisis.
+Prior developments during other recent outbreaks of emerging diseases, especially those caused by human coronaviruses (HCoV), have guided biomedical research into the behavior and treatment of this novel coronavirus infection.
+However, previous emerging HCoV-related disease threats were controlled much more quickly than SARS-CoV-2 through public health efforts (Figure @fig:csse-deaths).
+The scale of the COVID-19 pandemic has made the repurposing and development of pharmaceuticals more urgent than in previous coronavirus epidemics.
 
 #### Lessons from Prior HCoV Outbreaks
 
-<!--To Do: Update this figure in line with #736, #962, and 10.3390/ijerph17113973 (which explains why the SARS data don't start until March)-->
 ![
-**Cumulative global COVID-19 deaths since January 22, 2020.**
-{{csse_deaths}} COVID-19 deaths had been reported worldwide as of {{csse_date_pretty}} (Figure @fig:csse-deaths).
-Data are from the COVID-19 Data Repository by the Center for Systems Science and Engineering at Johns Hopkins University [@https://github.com/CSSEGISandData/COVID-19/tree/master/csse_covid_19_data/csse_covid_19_time_series].
-Previous emergent viral threats had a much lower death toll that was not felt on a such a global scale, and in most cases were controlled much more quickly.
-For example, SARS-CoV-1 was controlled within 9 months of its appearance, whereas SARS-CoV-2 remains a significant global threat almost 1.5 years later.
+**Cumulative global incidence of COVID-19 and SARS.**
+As of {{csse_date_pretty}}, {{csse_cases}} COVID-19 cases and {{csse_deaths}} COVID-19 deaths had been reported worldwide since January 22, 2020 (Figure @fig:csse-deaths).
+In contrast, a total of 8,432 cases and 813 deaths were reported for SARS from March 17 to July 11, 2003.
```

Comment:
Is in contrast required here, when lines 43 and 44 are deleted?

Reply:
```suggestion
A total of 8,432 cases and 813 deaths were reported for SARS from March 17 to July 11, 2003.
```